javascript-conditionals/main.js

```diff
@@ -0,0 +1,76 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number) {
+  return number < 5;
```

Comment:
While this code is more precise and I'd argue more preferable, we want to make sure to you conditionals for all of these functions. 

eg 
function isUnderFive(number) {
  if (condition to check) {
  // code to run
  } else {
    // code to run
  }
}